opennms-services/src/main/resources/META-INF/opennms/applicationContext-pollerd.xml

```diff
@@ -28,304 +29,997 @@
+          <!-- TODO: NMS-8217: Use a more sophisticated naming convention for this queue -->
+          <simple>activemq:Location-${body.location}.Poller.HttpsMonitor</simple>
+        </recipientList>
+      </route>
+
+      <route id="locationLocalhostHttpsMonitorRoute">
+        <from uri="activemq:Location-localhost.Poller.HttpsMonitor" />
+        <bean ref="httpsMonitor" />
+      </route>
+
+
+      <!-- ICMP Monitor -->
+      <route id="pollIcmpMonitor">
+        <from uri="direct:pollIcmpMonitor" />
+        <!-- Calculate an appropriate JMS timeout based on the task -->
+        <setHeader headerName="CamelJmsRequestTimeout">
+          <simple>40000</simple>
+        </setHeader>
+        <!-- Use recipientList so that we can have a dynamic 'to:' URI. -->
+        <setExchangePattern pattern="InOut"/>
+        <recipientList>
+          <!-- TODO: NMS-8217: Use a more sophisticated naming convention for this queue -->
+          <simple>activemq:Location-${body.location}.Poller.IcmpMonitor</simple>
+        </recipientList>
+      </route>
+
+      <route id="locationLocalhostIcmpMonitorRoute">
+        <from uri="activemq:Location-localhost.Poller.IcmpMonitor" />
+        <bean ref="icmpMonitor" />
+      </route>
+
+
+      <!-- Imap Monitor -->
+      <route id="pollImapMonitor">
+        <from uri="direct:pollImapMonitor" />
+        <!-- Calculate an appropriate JMS timeout based on the task -->
+        <setHeader headerName="CamelJmsRequestTimeout">
+          <simple>40000</simple>
+        </setHeader>
+        <!-- Use recipientList so that we can have a dynamic 'to:' URI. -->
+        <setExchangePattern pattern="InOut"/>
+        <recipientList>
+          <!-- TODO: NMS-8217: Use a more sophisticated naming convention for this queue -->
+          <simple>activemq:Location-${body.location}.Poller.ImapMonitor</simple>
+        </recipientList>
+      </route>
+
+      <route id="locationLocalhostImapMonitorRoute">
+        <from uri="activemq:Location-localhost.Poller.ImapMonitor" />
+        <bean ref="imapMonitor" />
+      </route>
+
+
+      <!-- JDBC Monitor -->
+      <route id="pollJDBCMonitor">
+        <from uri="direct:pollJDBCMonitor" />
+        <!-- Calculate an appropriate JMS timeout based on the task -->
+        <setHeader headerName="CamelJmsRequestTimeout">
+          <simple>40000</simple>
+        </setHeader>
+        <!-- Use recipientList so that we can have a dynamic 'to:' URI. -->
+        <setExchangePattern pattern="InOut"/>
+        <recipientList>
+          <!-- TODO: NMS-8217: Use a more sophisticated naming convention for this queue -->
+          <simple>activemq:Location-${body.location}.Poller.JDBCMonitor</simple>
+        </recipientList>
+      </route>
+
+      <route id="locationLocalhostJDBCMonitorRoute">
+          <from uri="activemq:Location-localhost.Poller.JDBCMonitor" />
+          <bean ref="jdbcMonitor" />
+      </route>
+
+
+      <!--JDBC Query Monitor  -->
```

Comment:
Can you clean up the whitespace from here down so that it matches the rest of the Camel routes? Thanks.